### PR TITLE
chore: Run tests on Node versions used in apps

### PR DESCRIPTION
**Pull request by gmaclennan**
Mapeo Mobile currently runs on Node v10.13.0
Mapeo Desktop (electron v6.1.7) runs on Node v12.4.0
Latest electron is Node v12.14.1

### CODE REVIEW

**Comment by okdistribute** [comment]
There's a flaky test! T_T

**Comment by noffle** [comment]
Looks like this has been broken since the upgrade to `mapeo-core@8`.

I'm seeing it failing because an `EPIPE` is triggered. It looks like sync finishes ok though, because `stream.goodFinish` is set in mapeo-core. I'm making a fix in mapeo-core to see if that fixes it..

**Comment by noffle** [comment]
Hey cool, that did it! This should be good to merge.

File: test/observations.js [hunk 1]
```diff
@@ -561,3 +560,9 @@ function testUpdateObservation (t, orig, update, expected, cb) {
     })
   })
 }
+
+function idSort (a, b) {
+  if (a.id < b.id) return -1
```
Comment (okdistribute) [inline test/observations.js:565]: woot

**Pull request merged**